---
Begin generate bug_id: Mockito_11
num_of_hunks: 3
use working dir: ./working/data/Mockito_11
Try to checkout Mockito_11
build dir: ['./working/data/Mockito_11/build/classes/main', './working/data/Mockito_11/build/classes/test']
init FileManager
trace file: src/org/mockito/internal/creation/DelegatingMethod.java
trace file: src/org/mockito/internal/creation/DelegatingMethod.java
trace file: src/org/mockito/internal/creation/DelegatingMethod.java
trying to replace old failing tests
new test num: 0
ori test num: 2
pattern: 000
patches are written to file
running tests...
used time: Finished in 343.93 seconds
Failing tests: 2
	org.mockito.internal.creation.DelegatingMethodTest::equals_should_return_true_when_equal
	org.mockito.internal.creation.DelegatingMethodTest::equals_should_return_true_when_self
pattern: 001
try to fix hunk: 2
replace from 60 with range 1 to
        return method.hashCode();

patches are written to file
running tests...
used time: Finished in 299.00 seconds
Failing tests: 2
	org.mockito.internal.creation.DelegatingMethodTest::equals_should_return_true_when_equal
	org.mockito.internal.creation.DelegatingMethodTest::equals_should_return_true_when_self
No new failing tests but only 1 hunk
pattern: 010
try to fix hunk: 1
insert before 56 with         }

patches are written to file
running tests...
seems testing failed, see output:
<OUTPUT>
<Common Compile Failed>
<ENDOUTPUT>
pattern: 100
try to fix hunk: 0
insert before 55 with         if (this == o) {
            return true;
        }
        if (o instanceof DelegatingMethod) {
            DelegatingMethod that = (DelegatingMethod) o;
            return method.equals(that.method);
        } else {

patches are written to file
running tests...
seems testing failed, see output:
<OUTPUT>
<Common Compile Failed>
<ENDOUTPUT>
pattern: 011
try to fix hunk: 1
insert before 56 with         }

try to fix hunk: 2
replace from 60 with range 1 to
        return method.hashCode();

patches are written to file
running tests...
seems testing failed, see output:
<OUTPUT>
<Common Compile Failed>
<ENDOUTPUT>
pattern: 101
try to fix hunk: 0
insert before 55 with         if (this == o) {
            return true;
        }
        if (o instanceof DelegatingMethod) {
            DelegatingMethod that = (DelegatingMethod) o;
            return method.equals(that.method);
        } else {

try to fix hunk: 2
replace from 60 with range 1 to
        return method.hashCode();

patches are written to file
running tests...
seems testing failed, see output:
<OUTPUT>
<Common Compile Failed>
<ENDOUTPUT>
pattern: 110
try to fix hunk: 0
insert before 55 with         if (this == o) {
            return true;
        }
        if (o instanceof DelegatingMethod) {
            DelegatingMethod that = (DelegatingMethod) o;
            return method.equals(that.method);
        } else {

try to fix hunk: 1
insert before 56 with         }

patches are written to file
running tests...
used time: Finished in 299.33 seconds
Failing tests: 0
Can independently fix ['org.mockito.internal.creation.DelegatingMethodTest::equals_should_return_true_when_equal', 'org.mockito.internal.creation.DelegatingMethodTest::equals_should_return_true_when_self']
select new bug
Pattern: 110
failing tests: org.mockito.internal.creation.DelegatingMethodTest::equals_should_return_true_when_equal@org.mockito.internal.creation.DelegatingMethodTest::equals_should_return_true_when_self
pattern: 111
try to fix hunk: 0
insert before 55 with         if (this == o) {
            return true;
        }
        if (o instanceof DelegatingMethod) {
            DelegatingMethod that = (DelegatingMethod) o;
            return method.equals(that.method);
        } else {

try to fix hunk: 1
insert before 56 with         }

try to fix hunk: 2
replace from 60 with range 1 to
        return method.hashCode();

patches are written to file
running tests...
used time: Finished in 264.00 seconds
Failing tests: 1
	org.mockito.verification.TimeoutTest::should_pass_even_if_first_verification_fails
EXCEPTION: Could not pass after all fixed, should check
Includes new failing tests, set to unusable
Find 1 new bugs
